.github/CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
-# Contributing to this project
+# Contributing to a Unicode Consortium repository
```

Comment:
Reflecting that this CONTRIBUTING.md file applies to all repositories.

The `##` section below will be copied into any CONTRIBUTING.md for projects (ICU, CLDR, ICU4X) that have their own CONTRIBUTING.md.

"this project" was always nebulous (what is a project?).